_partials/_caggs-intro.md

```diff
@@ -8,12 +8,19 @@ temperature readings taken every second, you can find the average temperature
 for each hour. Every time you run this query, the database needs to scan the
 entire table and recalculate the average every time.
 
-Continuous aggregate views are refreshed automatically in the background as new
-data is added, or old data is modified. Timescale tracks these changes to the
-dataset, and automatically updates the view in the background. This does not add
-any maintenance burden to your database, and does not slow down `INSERT`
-operations.
+Continuous aggregates are a kind of hypertable that is refreshed automatically
+in the background as new data is added, or old data is modified. Changes to your
+dataset are tracked, and the hypertable behind the continuous aggregate is
+automatically updated in the background. This does not add any maintenance
+burden to your database, and does not slow down `INSERT` operations.
```

Comment:
> This does not add any maintenance burden to your database, and does not slow down `INSERT` operations.

This is not true. It adds both maintenance burden (the cagg / materialized view has to be updated in regular intervals) and may slow down `INSERT` operations in resource constrained environments as you have more queries over the same hypertable. 

I would switch this to mentioning:
- They do not add any additional maintenance burden to the **application / user**. They are automatically kept up to date by Timescale, so the user has one less thing to think and worry about. Compare that to regular PostgreSQL materialized views, where the user must decide when to refresh the materialized view and execute a command to do so.
- They add far less maintenance burden to your database and have a much lower impact on insert and other operations compared to regular PostgreSQL materialized views. Due to the technology behind continuous aggregates, Timescale only incrementally updates the continuous aggregate. Compare that to regular PostgreSQL materialized views, where the whole view must be recreated from scratch, running very expensive queries over months or years of data, an operation that can become very time consume and expensive from a resource allocation perspective (e.g. multiple GBs of memory may be required to be allocated)